### DISCUSSION TITLE
TOOMANYREQUESTS on database DB

### DISCUSSION THREAD

**Discussion by DanielCastronovo** [aquasecurity/trivy#8034] [Bugs]
### Description

  FATAL   Fatal error     init error: DB error: failed to download vulnerability DB: database download error: OCI repository error: 1 error occurred:
        * GET https://ghcr.io/v2/aquasecurity/trivy-db/manifests/2: TOOMANYREQUESTS: retry-after: 634.968µs, allowed: 44000/minute

### Desired Behavior

do not download every time the download vulnerability DB

### Actual Behavior

download too many time the  vulnerability DB

### Reproduction Steps

```bash
trivy fs .
```

### Target

None

### Scanner

Vulnerability

### Output Format

JSON

### Mode

Standalone

### Debug Output

```bash
none
```

### Operating System

Linux

### Version

```bash
0.56.0
```

### Checklist

- [X] Run `trivy clean --all`
- [X] Read [the troubleshooting](https://aquasecurity.github.io/trivy/latest/docs/references/troubleshooting/)

**Comment by etaormm** [comment]
Update your trivy to v0.57.1

**Comment by simar7** [comment]
Please see https://github.com/aquasecurity/trivy/discussions/8009
